india_banking/overrides/journal_entry.py

```diff
@@ -0,0 +1,147 @@
+import frappe
+from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import (
+	get_accounting_dimensions,
+)
+from frappe.query_builder import DocType
+from frappe.query_builder.functions import Sum
+
+
+@frappe.whitelist()
+def make_payment_order(source_name, target_doc=None, args=None):
+	from frappe.model.mapper import get_mapped_doc
+
+	def update_bank_entry(source, target):
+		JournalEntryAccount = DocType("Journal Entry Account")
+		BankAccount = DocType("Bank Account")
+
+		select_field = [
+			"name",
+			"account",
+			"cost_center",
+			"project",
+			"debit as amount",
```

Comment:
Assuming debit and debit in account currency as same here? Can you add currency as INR as a validation in the query?